### ISSUE TITLE
logo

### ISSUE THREAD

**Issue by alexhsamuel** [alexhsamuel/supdoc#39]

**Comment by theodorosploumis** [comment]
Hi, I could help with this if you like. I am not a logo designer but I like creating logos and graphics for open source projects. Should I proceed?

Thanks in advanced!

**Comment by alexhsamuel** [comment]
Hi there. Thanks for the offer! There are some icons I pulled from around the web in `src/python/supdoc/html/static`, but if you'd like to design something original similar to these, that would be lovely.

Please note that at the moment, this project isn't getting too much attention fro me, however.

**Comment by theodorosploumis** [comment]
I was thinking about something like this. The colored python over a browser/window with the "#" included.

![supdoc_1-3](https://cloud.githubusercontent.com/assets/1315321/24520691/08a6d0b8-1592-11e7-8095-516f35a8a2ae.png)

**Comment by theodorosploumis** [comment]
Hi, any progress with this? Thanks! :-)

**Comment by alexhsamuel** [comment]
Hi! Thanks for the options, but it's not really what I had in mind. Please see the files I described above to get a sense for it.

**Comment by theodorosploumis** [comment]
The icon set has many icons. There is also a [carrot](https://github.com/alexhsamuel/supdoc/blob/master/src/python/supdoc/html/static/1475447724_Carrot.png) inside the folder. Would you prefer using a carrot like logo? Should I include the "Python" on the logo?

Thanks.

**Comment by alexhsamuel** [comment]
Yes, I thought the carrot would be a good play off the project name, and also something less generic. (Perhaps it's too obscure though?)

**Comment by theodorosploumis** [comment]
I could "recreate" the carrot icon with the python and the docs sign included. Let me try and come back soon.

**Comment by theodorosploumis** [comment]
Maybe something like this then?

![g4699](https://user-images.githubusercontent.com/1315321/26877573-fc4bf596-4b92-11e7-9c36-2fdaf48c8b83.png)
![supdoc_carrot_logo](https://user-images.githubusercontent.com/1315321/26877574-fc6e19f0-4b92-11e7-9eef-c72c35edfb42.png)
